clusterloader2/api/types.go

```diff
@@ -222,6 +224,14 @@ type RandomizedLoad struct {
 	AverageQPS float64 `json:"averageQps"`
 }
 
+
+// PoissonLoad defines a load with with Poisson arrival time
+// for a given rate parameter λ
+type PoissonLoad struct {
+	// RatePS specifies the possion rate parameter per second.
+	RatePS float64 `json:"ratePS"`
```

Comment:
I presume that the "PS" in the name `RatePS` stands for "Per Second" --- which is redundant with "rate".  I suggest naming the things that happen per second.  The usual name in this body of code seems to be "action".  So I suggest a name like "ActionsPerSecond".  Spelling things out is preferred in this community.

Also, since this is only probabilistic, there should be a word in here about that.  You might name this parameter "MeanRate" or "ExpectedActionsPerSecond" or something like that.  Being explicit about the units is preferred.